_posts/2023-03-13-introducing-security-analytics.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title:  "Introducing Security Analytics"
+authors:
+  - christophermoore
+date:   2023-03-13 12:15:00 -0700
+categories:
+  - releases
+meta_keywords: security analytics
+meta_description: Provides and overview of Security Analytics and how it fits in with the OpenSearch Project
+---
+
+# Introducing Security Analytics
+
+In January of 2023, one of the largest wireless carriers in the US revealed that hackers had compromised a company database of customer information through unauthorized use of an API and had acquired the personal details of as many as 37 million individuals, including names, addresses, phone numbers, and account numbers. Once identified, the company was able to halt the malicious activity within a day. However, investigations indicated that the data breach had likely occurred at the end of November the previous year, giving the hackers over a month to extract the information.
+
+Although unfortunate, occurrences like these now appear frequently in the media and have become almost commonplace in the modern world. With an ever-increasing volume of business data shared online and stored in the cloud, malicious attempts to disrupt companies’ business interests are now simply part of the reality of doing business. And in light of these threats, it’s therefore essential that companies have tools in place to detect and identify the signatures of possible attacks in real time so they can act to intervene as quickly as possible. To address these needs, OpenSearch now includes a package of pre-built features and functionality that allow businesses to quickly and easily roll out a security solution for confronting a full range of cyber threats. Security Analytics is this solution for OpenSearch, and it installs automatically with any distribution.
```

Comment:
" ... in the media ..." doesnt support the core argument. i would consider editing the lead line here to something like "In the modern world where devices are interconnected across virtual boundaries, it has become commonplace for such security attacks to occur."

Reply:
Should we say enterprises instead of business?